i18n-msg.html

```diff
@@ -46,6 +46,18 @@
       </body>
     </html>
 
+### Using Tokens
+
+Tokens can be used to pass dynamic content to a translated string.  For example, "Hello [name], how are you doing?" is a string that can be translated, but where the name should be left alone.  In this case, you would set the msgid to "Hello !name, how are you doing", and pass a variable {!name: Maria} as the tokens attribute.  The  json would look something like:
+
+  "Hello !name, how are you doing?": {
```

Comment:
This method of passing variables into translated strings is similar to how it is handled in Drupal 7.  Here is a link to the t() function, which basically behaves the same way:

https://api.drupal.org/api/drupal/includes%21bootstrap.inc/function/t/7